components/docs-chef-io/Makefile

```diff
@@ -4,7 +4,9 @@ SHELL=bash
 
 assets:
 	git submodule update --init --recursive
-	pushd chef-web-docs && git pull origin master && git reset --hard && pushd themes/docs-new && make assets && popd
+	git submodule foreach git pull origin master
+	git submodule foreach git reset --hard
```

Comment:
This change will update all the [submodules](https://github.com/habitat-sh/habitat/blob/master/.gitmodules) in this repo. It's probably not a huge deal since the other submodules haven't had a new release in years, but it could cause problems if they're updated. The existing line only affects chef-web-docs.

Reply:
reverted